src/app/clients/getClientParcelsData.ts

```diff
@@ -0,0 +1,69 @@
+import supabase from "@/supabaseClient";
+import { DatabaseError } from "@/app/errorClasses";
+import { Data } from "@/components/DataViewer/DataViewer";
+import { formatDatetimeAsDate, formatDatetimeAsTime } from "@/app/parcels/getExpandedParcelDetails";
+
+export type RawClientParcelsDetails = Awaited<ReturnType<typeof getRawClientParcelsDetails>>;
+type ClientParcelDetails = RawClientParcelsDetails[number];
+interface ParcelsDetail {
+    parcel_id: string;
+    collection_centre?: { name: string; acronym: string } | null;
+    collection_datetime: string | Date | null;
+    packing_datetime: string | null;
+    voucher_number?: string | null;
+}
+export const getExpandedClientParcelsDetails = async (
+    clientId: string
+): Promise<ExpandedClientParcelDetails[]> => {
+    const rawClientParcelsDetails = await getRawClientParcelsDetails(clientId);
+    const formattedList = rawClientParcelsDetails.map((parcel) => {
+        return rawDataToExpandedClientParcelsDetails(parcel);
+    });
```

Comment:
```suggestion
    const formattedList = rawClientParcelsDetails.map(rawDataToExpandedClientParcelsDetails);
```
you can simplify this expression :)